eiffel-vocabulary/EiffelServiceDeployedEvent.md

```diff
@@ -14,7 +14,7 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 --->
-
+ 
```

Comment:
The EiffelServiceDeployedEvent is to declare that  a previously allocated service instance ((declared by [EiffelServiceAllocatedEvent]) has been configured and deployed with required Artifacts by a machine/human.
 It implies that the service instance is ready to be started.

We could discuss more on data types.